internal/pkg/ccloudv2/flink_gateway.go

```diff
@@ -106,10 +107,65 @@ func (c *FlinkGatewayClient) GetStatementResults(environmentId, statementId, org
 		req = req.PageToken(pageToken)
 	}
 	resp, httpResp, err := req.Execute()
-	return resp, errors.CatchCCloudV2Error(err, httpResp)
+	return resp, makeFlinkError(err, httpResp)
 }
 
 func (c *FlinkGatewayClient) GetExceptions(environmentId, statementId, orgId string) (flinkgatewayv1alpha1.SqlV1alpha1StatementExceptionList, error) {
 	resp, httpResp, err := c.StatementExceptionsSqlV1alpha1Api.GetSqlV1alpha1StatementExceptions(c.flinkGatewayApiContext(), environmentId, statementId).OrgId(orgId).Execute()
-	return resp, errors.CatchCCloudV2Error(err, httpResp)
+	return resp, makeFlinkError(err, httpResp)
```

Comment:
This wording aligns better with our other error catching functions
```suggestion
	return resp, catchFlinkError(err, httpResp)
```